src/Workspaces/MSBuild/Core/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
```

Comment:
I would expect we could remove these Microsoft.Buld* package references now.

Reply:
I think we'll still have problems with the ILogger API points.